lib/hooks/useSdkv2.ts

```diff
@@ -36,27 +26,30 @@ export type UseSdkv2 = [
  * @returns UseSdkv2
  */
 export const useSdkv2 = (): UseSdkv2 => {
-  const store = useStore();
   const [sub, setSub] = useState<Subscription>();
   const [sdk, setSdk] = useState<Sdk<Context> | null>();
 
-  const id = identify(store);
+  const id = identify(
+    endpoints.map((e) => e.value),
+    graphQlEndpoint,
+  );
   const prevId = usePrevious(id);
 
   useEffect(() => {
-    if ((id && store.userStore.endpoint) || store.userStore.gqlEndpoint) {
+    const endpointVals = endpoints.map((e) => e.value);
+    if ((id && endpoints) || graphQlEndpoint) {
       if (sub && prevId && id !== prevId) {
         setTimeout(() => {
           init.cache.delete(prevId);
           sub.unsubscribe();
         }, 500);
       }
 
-      const sdk$ = init(store);
+      const sdk$ = init(endpointVals, graphQlEndpoint);
       //@ts-ignore todo: adjust type in sdk
```

Comment:
Can you link the type error you get here? Want to fix before merging.

Reply:
![image](https://user-images.githubusercontent.com/551965/226874415-f01c7d97-b129-42f1-8d0a-598b57646da9.png)